src/VisualStudio/Core/Def/Implementation/PullMemberUp/WarningDialog/PullMemberUpDialogWarning.xaml

```diff
@@ -0,0 +1,83 @@
+﻿<vs:DialogWindow
+             x:Uid="PullMemberUpWarningDialog" 
+             x:Class="Microsoft.VisualStudio.LanguageServices.Implementation.PullMemberUp.PullMemberUpDialogWarningxaml"
+             x:ClassModifier="internal"
+             x:Name="dialog"
+             xmlns:vs="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:u="clr-namespace:Microsoft.VisualStudio.LanguageServices.Implementation.Utilities"
+             xmlns:sys="clr-namespace:System;assembly=mscorlib"
+             mc:Ignorable="d" 
+             SizeToContent="WidthAndHeight"
+             WindowStartupLocation="CenterOwner"
+             Height="290"
+             Width="490"
+             Title="{Binding ElementName=dialog, Path=PullMembersUpTitle}">
```

Comment:
Sure! I'm thinking it will be a new topic under [Quick Actions](https://docs.microsoft.com/en-us/visualstudio/ide/quick-actions?view=vs-2017) that explains what the fix does. I can file an issue to create that doc once this is merged.